tests/printer/comments/pattern.res

```diff
@@ -58,6 +58,18 @@ let /* before */ {
 let /* b1 */ Blue /* b2 */ | /* c1 */ Red /* c2 */ = color
 let /* b1 */ Blue /* b2 */ | /* c1 */ Red /* c2 */ | /* d1 */ Green /* d2 */ = color
 
+// #462: Comments in nested Ppat_or must not disappear
+let _ = switch "anything" {
+// some comment
+| "first pattern"
+// first pattern
```

Comment:
Can you name the comments a bit more clear?
`// first pattern` seems to be above "second pattern".

Reply:
Done